relay-general/src/store/transactions/processor.rs

```diff
@@ -319,8 +319,13 @@ impl Processor for TransactionsProcessor<'_> {
             event.transaction_info.value_mut(),
         )?;
 
-        // Normalize transaction names for URLs transaction sources only.
-        if event.get_transaction_source() == &TransactionSource::Url && self.normalize_names {
+        // Normalize transaction names for URLs and Sanitized transaction sources.
+        // This in addition to renaming rules can catch some high cardinality parts.
+        if matches!(
+            event.get_transaction_source(),
+            &TransactionSource::Url | &TransactionSource::Sanitized
+        ) && self.normalize_names
+        {
```

Comment:
We could also run `normalize_transaction_name` before `apply_transaction_rename_rule`, would that result in the same name? Either way, we should think about what we put into `_meta` when both functions find something. Two separate `"rem"`s, maybe?

Reply:
If we run the normalization before the renaming rules, it can happen that the rules won't match anymore. Yeah, we might have to update the meta accordingly.